### PR TITLE
feature/MAMS-5-adding-new-address-and-getting-address

### PR DESCRIPTION
**Pull request by klezner**
Add feature/MAMS-5-adding-new-address-and-getting-address to develop

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,6 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+		</dependency>
```
Comment (MichalKulygin) [inline pom.xml]: too much empty lines

**Review by MichalKulygin — APPROVED** [review]
ok

File: pom.xml (hunk 1 above)
Comment (klezner) [reply pom.xml]: Fixed

**Review by KatBre — COMMENTED** [review]
Everything is ok. No unnecessary lines.

**Review by KatBre — APPROVED** [review]

**Pull request merged**